FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift

```diff
@@ -222,7 +222,7 @@ extension ThreadDetailsViewController {
                 input[index].rawMessage = updatedMessage
                 node.reloadSections(IndexSet(integer: index), with: .fade)
             } catch {
-                showToast("Could not mark message as read: \(error)")
+                showToast("Could not mark message as read: \(error.localizedDescription)")
```

Comment:
Localisation?

Reply:
can be spelled both ways

Reply:
I mean do we need to move this to our localisation file?

Reply:
I see. Sure - that never hurts.